helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/HelpDeskBotStories.java

```diff
@@ -108,4 +124,43 @@ public InjectableStepsFactory stepsFactory() {
         "**/*.story", "**/excluded*.story");
   }
 
+  private Room getQueueRoom() {
+    try {
+      return createRoom(Boolean.TRUE);
+    } catch (RoomException e) {
+      try {
+        return createRoom(Boolean.FALSE);
+      } catch (RoomException e1) {
+        throw new IllegalStateException("Couldn't create queue room.");
+      }
+    }
+  }
+
+  private Room createRoom(Boolean showHistory) throws RoomException {
+    SymRoomAttributes symRoomAttributes = new SymRoomAttributes();
+    symRoomAttributes.setViewHistory(showHistory);
+
+    String randomId = UUID.randomUUID().toString();
+    symRoomAttributes.setName("Queue Room" + randomId);
```

Comment:
Please include the white space between the name and the UUID. The same comment for the description.

Reply:
Done.